pkg/apis/componentconfig/types.go

```diff
@@ -230,7 +230,7 @@ type KubeletConfiguration struct {
 	// be rejected.
 	LowDiskSpaceThresholdMB int `json:"lowDiskSpaceThresholdMB"`
 	// How frequently to calculate and cache volume disk usage for all pods
-	VolumeStatsAggPeriod unversioned.Duration `json:volumeStatsAggPeriod`
+	VolumeStatsAggPeriod unversioned.Duration `json:"volumeStatsAggPeriod"`
```

Comment:
For this one, I'm not sure. It doesn't seem like a normal API type, but if it's user visible, I would imagine so. Again, I defer to @bgrant0607 on these.